packages/storybook-vue/config/main.ts

```diff
@@ -35,6 +28,7 @@ const config: StorybookConfig = {
       css: {
         preprocessorOptions: {
           scss: {
+            silenceDeprecations: ['import'],
```

Comment:
Zou je een comment willen toevoegen waar dit voor is, en eventueel wanneer we het weg kunnen halen?